src/utils/object.ts

```diff
@@ -0,0 +1,14 @@
+export const objectKeys = <T extends { [_: string]: any }, K extends keyof T>(
+  obj: T,
+): K[] => Object.keys(obj) as K[];
+
+export const objectValues = <T extends { [_: string]: any }, K extends keyof T>(
+  obj: T,
+): T[K][] => Object.values(obj);
+
+export const objectEntries = <
+  T extends { [_: string]: any },
+  K extends keyof T,
+>(
+  obj: T,
```

Comment:
What's the advantage to using `.keys`, `.values` and `.entries` like this?

Reply:
+1, we don't need typings for native methods